doc/near/bts.md

```diff
@@ -0,0 +1,177 @@
+| account_id | string | should be a valid account id |
+| coin_name | string |  |
+
+**CLI Command** 
+```console
+NEAR_ENV=testnet near view <BTS> balance_of '{"account_id": "<ACCOUNT ID>", "coin_name": "<Coin Name>"}'
+```
+
+#### Refundable Balance
+**Method** 
+- refundable_balance_of
+
+| Parameters | Type | Info |
+|:---------|:--------|:--------|
+| account_id | string | should be a valid account id |
+| coin_name | string |  |
+
+**CLI Command** 
+```console
+NEAR_ENV=testnet near view <BTS> refundable_balance_of '{"account_id": "<ACCOUNT ID>", "coin_name": "<Coin Name>"}'
+```
+
+#### Locked Balance
+**Method** 
+- locked_balance_of
+
+| Parameters | Type | Info |
+|:---------|:--------|:--------|
+| account_id | string | should be a valid account id |
+| coin_name | string |  |
+
+**CLI Command** 
+```console
+NEAR_ENV=testnet near view <BTS> locked_balance_of '{"account_id": "<ACCOUNT ID>", "coin_name": "<Coin Name>"}'
+```
+
+### Deposit
+
+#### Deposit NEAR to BTS
+**Method** 
+- deposit
+
+**CLI Command** 
+```console
+NEAR_ENV=testnet near call <BTS> deposit --amount <AMOUNT in NEAR> --accountId <ACCOUNT ID>
+```
+#### Deposit Cross-Chain Native Coin to BTS
+**Method** 
+- ft_transfer_call
+
+| Parameters | Type | Info |
+|:---------|:--------|:--------|
+| receiver_id | string | should be a valid account id |
+| amount | string |  |
+| msg | string |  |
+
+**CLI Command** 
+```console
+NEAR_ENV=testnet near call <NEP141 Contract> ft_transfer_call '{"receiver_id": "<BTS>", "amount": "<AMOUNT>", "msg": ""}' --accountId <ACCOUNT ID> --amount <1 yoctoNEAR in highest Denomination ie 0.000000000000000000000001>
+```
+
+### Withdraw
+**Method** 
+- withdraw
+
+| Parameters | Type | Info |
+|:---------|:--------|:--------|
+| coin_name | string |  |
+| amount | string |  |
+
+**CLI Command** 
+```console
+NEAR_ENV=testnet near call <BTS> withdraw '{"coin_name": "<COIN NAME>", "amount":"<Amount in lowest Denomination>"}' --amount <1 yoctoNEAR in highest Denomination ie 0.000000000000000000000001> --gas 300000000000000 --accountId <ACCOUNT ID>
+```
+
+### Fee
+
+#### Get Fee
+**Method** 
+- get_fee
+
+| Parameters | Type | Info |
+|:---------|:--------|:--------|
+| coin_name | string |  |
+| amount | string |  |
+
+**CLI Command** 
+```console
+NEAR_ENV=testnet near view <BTS> get_fee '{"coin_name": "<Coin Name>", "amount": "<Amount in lowest Denomination, For NEAR in yoctoNEAR ie 1 NEAR = 1^24 yoctoNEAR>"}'
+```
+
+### Transfer
+**Method** 
+- transfer
+
+| Parameters | Type | Info |
+|:---------|:--------|:--------|
+| coin_name | string |  |
+| destination | string | valid btp address |
+| amount | string |  |
+
+**CLI Command** 
+```console
+NEAR_ENV=testnet near call <BTS> transfer '{"coin_name": "<Coin Name>", "destination": "btp://<Network>/<Address>", "amount": "<Amount in lowest Denomination, For NEAR in yoctoNEAR ie 1 NEAR = 1^24 yoctoNEAR>"}' --gas 300000000000000 --accountId <ACCOUNT ID>
+```
+
+
+## Usage
```

Comment:
These usages are not clear enough how to use because there are input parameters with no examples of what those should be. Please add the real-world usage for these commands instead of linking to the CLI commands.